src/components/projecteditor/welcome/index.js

```diff
@@ -32,12 +32,12 @@ export default class Welcome extends Component {
             <div class={style.centerVertically}>
                 <div class={style.textWrap}>
                     <h1>Superblocks Lab</h1>
-                    <h2>Buidl your thing</h2>
+                    <h2>Build your thing</h2>
```

Comment:
```suggestion
                    <h2>Buidl your thing</h2>
```
The reason is to follow this crypto thing about misspelling this word  (as done with hodl)

Reply:
Ok, I'll fix that.